MCprep_addon/spawner/mcmodel.py

```diff
@@ -376,7 +387,7 @@ def add_model(
 			# Give slight offset by normal for overlay geometry
 			if face_mat == "#overlay":
 				bmesh.ops.translate(bm, verts=face.verts,
-						    vec=0.02 * face.normal)
+						    vec=0.005 * face.normal)
```

Comment:
I think we should test and see if this causes z-fighting issues

Reply:
It's already quite small gap.
![image](https://github.com/user-attachments/assets/a63ecb1d-2c51-480b-bf9c-f486b79c3767)

Reply:
Yeah as you noted in Discord it also causes z-fighting at far distances. Combining textures with Numpy might be the best option then